testing/mpdecimal/APKBUILD

```diff
@@ -0,0 +1,31 @@
+# Contributor: Stefan Stutz <stutz@pm.me>
+# Maintainer: Stefan Stutz <stutz@pm.me>
+pkgname=mpdecimal
+pkgver=2.4.2
+pkgrel=0
+pkgdesc="libmpdec is a complete implementation of the General Decimal Arithmetic Specification"
+url="http://www.bytereef.org/mpdecimal/index.html"
+arch="all"
+license="BSD"
+depends=""
+makedepends=""
+install=""
+subpackages=""
```

Comment:
<pre>>>> WARNING: mmpdecimal* Found /usr/share/doc but package name doesn't end with -doc</pre>
You need to have a `$pkgname-doc` subpackage